crates/floresta-wire/src/p2p_wire/node.rs

```diff
@@ -639,12 +642,21 @@ where
     }
 
     async fn maybe_open_connection(&mut self) -> Result<(), WireError> {
+        // If the user passes in a `--connect` cli argument, we only connect with
+        // that particular peer.
+        if self.fixed_peer.is_some() && !self.peers.is_empty() {
```

Comment:
Do we want this to be an exclusive or inclusive connection?
Meaning: if we pass a fixed peer, do we want to be sure it's being connect to?, or we want to exclusively connect to it?

Reply:
I'm mimicking Core's behavior, it connects to **only** this peer. The `inclusive` case is another option, we can add this later.